deploy/Makefile

```diff
@@ -55,10 +55,10 @@ postgres_logs:
 postgres_psql:
 	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml --file docker-compose.override.yml --profile postgres exec postgres psql -h localhost -U ${POSTGRES_USER} postgres
 
-create_db_setup_script:
+create_db_setup_script:  # NOTE: This script assumes we have an existing instance to setup the DB, maybe we can rather just let this run locally?
 	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml --file docker-compose.override.yml run web /bin/bash -c "ZEN_DB_LOG_ONLY=1 ./scripts/create_deployment_database.sh ${POSTGRES_HOST} ${INSTANCE_DB_NAME} ${POSTGRES_USER}"
 
-upgrade_database:
+upgrade_database:  # Similar to above
```

Comment:
Not quite the same:
- assumes we have an existing instance to setup the DB - yes ; 
- rune locally- no ;

Reply:
Well, this is slightly different. The main thing this is tied to web and cannot be done independently. However doesn't this happen on initial server startup?

Reply:
Oh true - it does - so one could skip this?

Maybe reach out to @AbbyBeeler I think she wrote the original documentation, and also mentioned that it happens on startup anyway.

Would simplify documentation etc. just leaving this step out.

Reply:
It should be done on server startup automatically, the issue just comes from the ordering of commands. For example, right now the README has you run `upgrade_database` and then create a new user. You can't run create a user until those tables have been created. So we could remove this and then shift the create user step to after the web server has been started.

@Ideonvn does AQT work for you? There's nowhere in the README that says to run `populate_data_models` and AQT won't load unless there's at least one dimension and field in the database tables, so I would expect not. And then a similar note where `populate_data_models` can't be run until the database has been upgraded.

Reply:
@AbbyBeeler it won't load initially, but I got it to load. Best answer here is just experience. I had this struggle before and I think you showed me what to do.

I initially manually inserted some records into the DB trying to get the wiki sample data to work. Alas I just took a snipped form the covid data in zen-internal. Ingested that and ran `populate_data_models`. AQT works, but it still returns no data (I think this is because of the way I was ingesting).

Reply:
Okay we should make sure instructions regarding `populate_data_models` are added to the README then